.circleci/config.yml

```diff
@@ -294,50 +302,50 @@ workflows:
         #       tags:
         #         only: /.*/
         - build-www:
-            context: org-global
+            context: aws
```

Comment:
What is this "context" and why did we change it? Does it mean anything?

Reply:
Is this a CircleCI thing? I see `context: aws` is also used in these examples: https://circleci.com/developer/orbs/orb/circleci/aws-cli

Reply:
a context is like a namespace for environment variables. 
'org-global' is apparently the context NH has in their repo setup. 
we don't have 'org-global', 
'aws' context was created by me to hold parameters related to AWS

Reply:
Ok makes sense, thanks :+1: